test/pp-output-cases/filter-tests.js

```diff
@@ -1,3 +1,27 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
@jeremy-rifkin this file you contributed 2 months ago was missing a license header. Are you ok with this copyright attribution?

Reply:
Yes thank you!